### PR TITLE
Ako/ use env to prevent script injection

### PR DESCRIPTION
**Pull request by ali-hosseini-deriv**
Changes:

This is to prevent the script injection on PR creation step.

## Type of change

-   [x] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [x] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: .github/workflows/pre-generate-preview-link.yml [hunk 1]
```diff
@@ -6,24 +6,29 @@ permissions:
...
         steps:
             - name: Retrieve PR information
+              env:
+                  EVENT_NUMBER: ${{ github.event.number }}
+                  EVENT_USERNAME: ${{ github.event.pull_request.user.login }}
               run: |
                   mkdir -p ./pr
-                  echo ${{ github.event.number }} > ./pr/NR
-                  echo ${{ github.event.pull_request.user.login }} > ./pr/USERNAME
-                  echo ${{ github.head_ref }} > ./pr/BRANCHNAME
```
Comment (balakrishna-deriv) [inline .github/workflows/pre-generate-preview-link.yml:23]: We still need branch name for smoke test to work fine. It's missing in the new lines.

File: src/translations/ach.json [hunk 2]
```diff
@@ -4545,4 +4545,4 @@
   "-1165835520": "crwdns459708:0crwdne459708:0",
   "-651384976": "crwdns459712:0crwdne459712:0",
   "-1033881248": "crwdns2198943:0crwdne2198943:0"
-}
```
Comment (balakrishna-deriv) [inline src/translations/ach.json:4548]: Can we revert these changes?

File: .github/workflows/pre-generate-preview-link.yml (hunk 1 above)
Comment (ali-hosseini-deriv) [reply .github/workflows/pre-generate-preview-link.yml:23]: my miss, fixed.

File: src/translations/ach.json (hunk 2 above)
Comment (ali-hosseini-deriv) [reply src/translations/ach.json]: done

**Review by balakrishna-deriv — APPROVED** [review]

**Review by habib-deriv — APPROVED** [review]
